projects/airpuff/index.js

```diff
@@ -11,6 +12,19 @@ const contractAbis = {
   getMswBalance: "function getAllEigeinPieCycleDepositAmounts() external view returns (uint256)",
   getUnderlyingPrice: "function getUnderlyingPrice(address cToken) view returns (uint256)",
 };
+async function fetchPendlePrice(assetAddress) {
+  // API endpoint template with variable part for the asset address
+  const url = `https://api-v2.pendle.finance/core/v1/1/assets/${assetAddress}`;
+
+  try {
+    const response = await axios.get(url);
+    const accValue = response.data.price.acc;
+    return accValue; // Return the "acc" value for further use
+  } catch (error) {
```

Comment:
hi, can you skip catching error part? we prefer the adapter to fail, this way we know there is a problem

Reply:
also, think we price pendle tokens in our coins api

Reply:
thanks for the review, I've changed the api endpoint to DefiLlama api.